react-native-community/react-native-maps
# Add onDoublePress callback

### Does any other open PR do the same thing?

Adds `onDoublePress` callback to Google Maps on Android and Apple Maps on iOS. Previous related work in #2692 and #2935. Created this PR as an alternative to #2692 with the reasoning that 

- the API is more explicit; adds a new `onDoublePress` callback instead of mixing multiple gestures to be handled by a single callback.
- the PR only does a single thing rather than multiple changes at once
- the previous PR has stalled

### What issue is this PR fixing?
Adds `onDoublePress` callback to Google Maps on Android and Apple Maps on iOS.

This can be used for multiple things, but primarily, to know when the user has interacted with the map by double tapping the screen.

### How did you test this PR?

Tested with emulator on Android and simulator on iOS.

1. Add onPress and onDoublePress callback handlers to the map
2. Observe that tapping once fires only onPress once, and tapping twice fires only onDoublePress once. 

I did not include changes for Google Maps on iOS since this does not match our use case. Let me know if it's a requirement to get it merged.
